foundations/config.yaml

```diff
@@ -6,17 +6,27 @@ metadata:
 data:
   foundations.yaml: |
     foundations:
-      clusterName: "bocchi"
+      clusterName: "cru"
       intranetDomain: "bocchi.rocks" # results in cluster address being `https://k8s.bocchi.rocks`
-    #  disable:
-    #    cilium: true
-    #    longhorn: true
-    #    openebs: true
-    #    ingress: true
-    #    klum: true
+      disable:
+       cilium: true
+       longhorn: true
+       openebs: true
+       ingress: true
+       klum: true
     #  cloud:
     #     region: ber1
     #     publicIPs:
     #       - cidr: 192.168.181.1/32
     #       - cidr: 192.168.181.2/32
-    #     metricsCollectorUri: "http://<cortex>/prometheus/api/v1/push"
+        metricsCollectorUri: "https://cortex.0x.pt/prometheus/api/v1/push"
+---
+# If you enable metricsCollector, and it's properly configured
+# for basic_auth with that cluster name & a password, you can create this secret
+apiVersion: v1
+kind: Secret
+metadata:
+  namespace: foundations
+  name: metrics-collector-secret
+stringData:
+  password: "EERYPKFKCFNAODKCRBMLNFUY"
```

Comment:
Uh oh

Reply:
yeah i'm not 100% done yet (and yeah I've also rotated it, oops)